flips/20210916-nft-metadata.md

```diff
@@ -0,0 +1,338 @@
+# Non-Fungible Token Metadata
+
+| Status        | Proposed |
+:-------------- |:---------------------------------------------------- |
+| **FLIP #**    | [636](https://github.com/onflow/flow/pull/636)                                                                                       |
+| **Author(s)** | Bjarte Karslen (bjartek@find.xyz), Deniz Mert Edincik (Deniz@edincik.com), Brian Dilley (briandilley@briandilley.com) |
+| **Sponsor**   | Peter Siemens (peter@dapperlabs.com)                                                                                                 |
+| **Updated**   | 2021-12-06                                                                                                                           |
+
+
+## Objective
+This proposal will make it possible to create generic solutions that will interoperate 
+through views and not hard-coded NFT types. Want to create a marketplace? If you know 
+how to display a certain Type that defines a sellable Item you can do so. And it will 
+work for any contract and any resource, not just a single one. 
+
+## Motivation
+Currently there is no standard for defining NFT/resource metadata. At best, most implementations 
+employ a simple `{String: AnyStruct}` dictionary. Due to the various methods that NFT 
+authors use it’s hard / impossible to create applications that use NFTs whether it be 
+for something as simple as viewing them or as complex as integrating them into a game 
+or application.
+
+If you want to create an Auction contract on Flow today you have to hardcode all the NFTs 
+you want to support. This will often lead to duplicated contracts and code and just lots 
+of things to maintain. This standard will enable you to create a single Auction contract 
+and define the Type of views you need to expose in your NFT in order to support the contract. 
+So it will enable a design-by-contract pattern where the creator of a contract can define 
+the views it requires and an NFT can tell what views they export. 
+
+All NFTs and contracts using NFTs on Flow are affected by this problem. This is most 
+visible in the number of contracts that Viv3 needs in order to support all their different 
+sellable items. With all the oncoming marketplaces/auction houses and things like that 
+coming to Flow a good standard for NFTs will help everybody. 
+
+## User Benefit
```

Comment:
FLOW developers' time will be saved, they won't have to integrate N contracts into their application, meaning they can use their dev cycles for other things that will indirectly benefit the user.

Is it worth mentioning this indirect user benefit?

Reply:
It could be, can you make a suggestion?

Reply:
Looks like this was resolved below, nice 👍